pkg/apis/pipeline/v1alpha1/taskrun_types.go

```diff
@@ -195,6 +195,7 @@ func (tr *TaskRunStatus) InitializeCloudEvents(targets []string) {
 type StepState struct {
 	corev1.ContainerState
 	Name string `json:"name,omitempty"`
+	Container string `json:"container,omitempty"`
```

Comment:
How about `ContainerName` ?

Reply:
Agreed - `Container` isn't specific enough. I prefer the more explicit `ContainerName`.

Reply:
good call, thanks fellow OWNERS :D

Reply:
:+1:

Reply:
yeah makes sense :+1: